website/docs/addon_core_settings.md

```diff
@@ -99,6 +99,16 @@ Applicable context filters:
 
 (This image shows use case where `render` anatomy template is used for products of families ['review, 'render', 'prerender'], `publish` template is chosen for all other.)
 
+### Filter creator profiles
+
+This provides a way how to limit list of provided Creators in Publisher based on current context. This should
+help artists decision which creator they should use.
+
+If you want to limit which creators should be shown, provide some profile, based on host name and task info.
+Put creator identifier to `Allowed Creator Identifiers`, in most cases it should be same value as resulting product type (lower cased).
```

Comment:
Hey,
I proof read and enhanced the writing using AI. Let me know what do you think. 

```suggestion
This feature offers settings that narrow down the list of creators appearing in Publisher according to the specific context, thereby making it easier for artists to choose the right creator for their work.

Each profile consists of 
- A list of host names.
- Task Types drop down menu.
- A list of task names, which can serve as an alternative to task types.
- A list of Creator Identifiers, where you can add the identifiers of the creators you want to be displayed.
```